R/pretty-print.R

```diff
@@ -113,7 +113,8 @@ transformStyles <- function (trans, cats) {
 
     # make a list of styles to apply
     styles <- lapply(all_labs, function (lab) {
-        if (is.Subtotal(lab)) {
+        if (is.Subtotal(lab) | is.SummaryStat(lab)) {
```

Comment:
Not a big issue, but if we're going to continue to keep adding new transforms it might be worth keeping track of the styles in a more streamlined way like a switch statement or a list.